docs/gen-docs/kyma_alpha_deploy.md

```diff
@@ -16,12 +16,24 @@ kyma alpha deploy [flags]
 
 ```bash
       --cancel-timeout duration   Time after which the workers' context is canceled. Pending worker goroutines (if any) may continue if blocked by a Helm client. (default 15m0s)
-  -c, --components string         Path to a YAML file with component list to override. (required)
+  -c, --components string         Path to a KYMA components file (default "workspace/kyma/installation/resources/components.yaml")
+  -d, --domain string             Domain used for installation. (default "local.kyma.dev")
       --helm-timeout duration     Timeout for the underlying Helm client. (default 6m0s)
-  -o, --overrides string          Path to a YAML file with parameters to override.
+  -o, --overrides string          Path to a JSON or YAML file with parameters to override.
+  -p, --profile string            Kyma deployment profile. Supported profiles are: evaluation, production
       --quit-timeout duration     Time after which the deployment is aborted. Worker goroutines may still be working in the background. This value must be greater than the value for cancel-timeout. (default 20m0s)
-  -r, --resources string          Path to Kyma resources folder. (required)
+  -r, --resources string          Path to a KYMA resource directory. (default "workspace/kyma/resources")
+  -s, --source string             Installation source. 
+                                  	- To use a latest release, write "kyma alpha deploy --source=latest".
+                                  	- To use a specific release, write "kyma alpha deploy --source=1.17.1".
+                                  	- To use the master branch, write "kyma alpha deploy --source=master".
+                                  	- To use a commit, write "kyma alpha deploy --source=34edf09a".
+                                  	- To use a pull request, write "kyma alpha deploy --source=PR-9486".
+                                  	- To use the local sources, write "kyma alpha deploy --source=local". (default "latest")
+      --tls-cert string           TLS certificate for the domain used for installation. The certificate must be a base64-encoded value.
+      --tls-key string            TLS key for the domain used for installation. The key must be a base64-encoded value.
       --workers-count int         Number of parallel workers used for the deployment. (default 4)
+  -w, --workspace string          Path used to download Kyma sources. (default "workspace")
```

Comment:
Can we unify the punctuation situation here? Some of these sentences end in a period and some don't. This is a bit sloppy, and since this documentation is deployed to our website ([here](https://kyma-project.io/docs/cli/), we should tidy it up. 
I am aware that this is auto-generated documentation, so could you introduce the corrections in the source from which it is generated?
I need to split these corrections into a few comments. Please see the other comments.

Reply:
Good point, sure that can be aligned. Will fix it.